package.json

```diff
@@ -174,7 +175,7 @@
     "cozy-authentication": "1.19.2",
     "cozy-bar": "7.10.0",
     "cozy-ci": "0.4.1",
-    "cozy-client": "9.7.0",
+    "cozy-client": "9.8.0",
```

Comment:
why not 10.X ? :D

Reply:
I just needed the `createMockClient` function for this PR. I didn't want to take the time to deal with breaking change or the risk of breaking something in this PR. I'm going to update it to the latest version in another one.